src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -458,7 +471,8 @@ private static void addConnectorConfigs(ConfigDef configDef) {
             CONNECTOR_GROUP,
             ++order,
             Width.SHORT,
-            DATA_STREAM_TYPE_DISPLAY
+            DATA_STREAM_TYPE_DISPLAY,
+            DATA_STREAM_TYPE_RECOMMENDER
```

Comment:
we don't need to put this in static variables, we can just do it inline
```suggestion
            Recommenders.enumValues(...)
```